Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM rocker/rstudio:4.3.2
+
+# no login required
+ENV DISABLE_AUTH=true
+
+RUN apt-get -y update && \
+apt-get -y install libpng-dev \
+python3 \
+python3-pip \
+python3-venv \
+python3-boto3 \
+python-is-python3 && \
+apt-get clean
+
+# This is needed to make reticulate work
+# https://github.com/rstudio/reticulate/issues/1509
+RUN python3 -m pip install virtualenv
+
+
+# Install essential R packages
+
+# Install tidyverse, devtools, BiocManager
+RUN R -e "install.packages(c('tidyverse','devtools','BiocManager'))"
+
+# Install synapser
+# environment variable needed to communicate with the embedded python and install boto3 dependency
+RUN R -e "Sys.setenv(SYNAPSE_PYTHON_CLIENT_EXTRAS='boto3'); install.packages('synapser', repos=c('http://ran.synapse.org', 'http://cran.fhcrc.org'))"
+
+# Let rstudio have sudo access without having to enter a password
+RUN echo 'rstudio   ALL=(ALL) NOPASSWD:ALL' | sudo EDITOR='tee -a' visudo
```

Comment:
In general this doesn't seem like a good practice to me because i think it will just allow users more opportunities to get into bad situations which may require Sage IT rescue.   

The typical and safer way to get sudo access is to just use the `sudo` command, shouldn't the user be expected to know that?

Reply:
@zaro0508 
> The typical and safer way to get sudo access is to just use the sudo command

That's what this change does:  It allows the `rstudio` user to have passwordless `sudo` access.  It doesn't mean they don't type, `sudo`.  Here's an example (after this change was made):

```
> rstudio@e5bb73e0892e:~$ whoami
rstudio
> rstudio@e5bb73e0892e:~$ apt-get update
Reading package lists... Done
E: Could not open lock file /var/lib/apt/lists/lock - open (13: Permission denied)
E: Unable to lock directory /var/lib/apt/lists/
rstudio@e5bb73e0892e:~$ sudo apt-get update
Hit:1 http://ports.ubuntu.com/ubuntu-ports jammy InRelease
Get:2 http://ports.ubuntu.com/ubuntu-ports jammy-updates InRelease [119 kB]
Hit:3 http://ports.ubuntu.com/ubuntu-ports jammy-backports InRelease
Get:4 http://ports.ubuntu.com/ubuntu-ports jammy-security InRelease [110 kB]
Get:5 http://ports.ubuntu.com/ubuntu-ports jammy-updates/multiverse arm64 Packages [28.4 kB]
Get:6 http://ports.ubuntu.com/ubuntu-ports jammy-updates/universe arm64 Packages [1,236 kB]
Get:7 http://ports.ubuntu.com/ubuntu-ports jammy-updates/restricted arm64 Packages [1,097 kB]
Get:8 http://ports.ubuntu.com/ubuntu-ports jammy-updates/main arm64 Packages [1,409 kB]
Get:9 http://ports.ubuntu.com/ubuntu-ports jammy-security/main arm64 Packages [1,140 kB]
Get:10 http://ports.ubuntu.com/ubuntu-ports jammy-security/multiverse arm64 Packages [24.0 kB]
Get:11 http://ports.ubuntu.com/ubuntu-ports jammy-security/universe arm64 Packages [975 kB]
Get:12 http://ports.ubuntu.com/ubuntu-ports jammy-security/restricted arm64 Packages [1,083 kB]
Fetched 7,221 kB in 4s (1,607 kB/s)                  
Reading package lists... Done
```

Reply:
Makes it easier to install system dependencies (else you have to go back to AWS console to get shell access).